hmpps-prisoner-search-indexer/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/services/PrisonerSynchroniserService.kt

```diff
@@ -52,6 +53,71 @@ class PrisonerSynchroniserService(
     return prisoner
   }
 
+  internal fun reindexUpdate(ob: OffenderBooking, eventType: String): Boolean {
+    val summary = prisonerRepository.getSummary(ob.offenderNo, SyncIndex.RED)
+    val isChanged = summary?.run {
+      val restrictedPatient = runCatching { getRestrictedPatient(ob) }
+
+      val prisoner = Prisoner().translate(
+        existingPrisoner = summary.prisoner,
+        ob = ob,
+        incentiveLevel = Result.failure(Exception()),
+        restrictedPatientData = restrictedPatient,
```

Comment:
Slightly hacky, but setting incentiveLevel to failure leaves it blank. Update leaves it unchanged in the index